test/cypress/integration/widgets/blueprint_num_spec.ts

```diff
@@ -0,0 +1,51 @@
+describe('Number of Blueprints widget', () => {
+    const widgetId = 'blueprintNum';
+
+    function clickOnWidget() {
+        cy.get('.statistic').click();
+    }
+
+    function verifyUrl(expectedPageId: string) {
+        cy.location('pathname').should('be.equal', `/console/page/${expectedPageId}`);
+    }
+
+    function setWidgetConfiguration(pageToOpenOnClick: string) {
+        cy.editWidgetConfiguration(widgetId, () => {
+            cy.setSingleDropdownValue('Page to open on click', pageToOpenOnClick);
+        });
+    }
+
+    describe('Tests without mockLogin', () => {
+        before(() => cy.activate().login().addWidget(widgetId));
+
+        it('Opens the default page on click', () => {
+            clickOnWidget();
+            verifyUrl('blueprints');
+        });
+    });
```

Comment:
I think we can merge it as there's no need to have `describe` section when there's just one test case inside IMHO.

```suggestion
    it('opens the default page on click', () => {
        cy.activate().login().addWidget(widgetId);
        clickOnWidget();
        verifyUrl('blueprints');
    });
```

General:

Strings in `describe` and `it` statements are intended to create a sentence when concatenated. See: https://jenkins.cloudify.co/blue/organizations/jenkins/Stage-UI-System-Test/detail/Stage-UI-System-Test/1445/tests . For example "License Management is providing License Page that fetches license on page load" comes from such code:
```
describe('License Management', () => {
    // ...
    describe('is providing License Page that', () => {
        // ...        
        it('fetches license on page load', () => {
            // ...
```